modules/versioning.nix

```
@@ -169,6 +169,23 @@ let
             This includes the final database size (0.4) plus some extra storage (0.2).
       '';
     }
+    {
+      version = "0.0.57";
+      condition = config.services.liquidd.enable && config.services.liquidd.prune == 1000;
```

```suggestion
      condition = config.services.liquidd.enable;
```
With the new release, `prune = 1000` is not set anymore, so this condition will be `false` for `secure-node.nix` users.
One could negate the `prune` condition (`config.services.liquidd.prune != 1000`), but I think this is a bit too clever.

---

Of course... good catch. Should we add a readonly enable option to secure-node.nix to be able to detect this better in the future?

---

Yes, great idea. Can be implemented like this:
```
  # in secure-node.nix
  options = {
    # Used by ../versioning.nix
    nix-bitcoin.secure-node-preset-enabled = {};
  };

  # in versioning.nix
  condition = config.services.liquidd.enable && config.nix-bitcoin ? secure-node-preset-enabled;
```